authority/authority.go

```diff
@@ -684,6 +640,50 @@ func (a *Authority) init() error {
 		return err
 	}
 
+	// Check if a KMS with decryption capability is required and available
```

Comment:
Why was this moved down here?

Reply:
It depends on the most recent state of the provisioners, specifically SCEP provisioners (the only ones using RSA decrypters). When they're (re)loaded from the database the provisioners in the authority configuration get updated in-memory. If checked before the reload, the list of provisioners would be empty, or would contain (potentially) outdated info.

This essentially was an existing bug that I found when testing this.